uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/absence-tab.js

```diff
@@ -0,0 +1,27 @@
+(function (CRM, require) {
+  CRM.vars.leaveAndAbsences = {
+    baseURL: CRM.absenceTabApp.path
+  };
+
+  var srcPath = CRM.vars.leaveAndAbsences.baseURL + '/js/angular/src/leave-absences';
```

Comment:
Why all these different steps? Can't you simply define `CRM.vars.leaveAndAbsences.baseURL` in the backend?